test-files/cmd/slow/ConflateCmdTest.sh.stdout

```diff
@@ -1,48 +1,95 @@
+11:44:47.352 INFO  ...ighway/HighwayRfClassifier.cpp( 305) factorLabels: [4]{EdgeDistanceRmseAggregator, EdgeDistanceSigma_Aggregator, hoot__AngleHistogramExtractor, hoot__WeightedMetricDistanceExtractor_way_agg__MeanAggregator_point_agg__RmseAggregator}
+11:44:47.356 INFO  ...ighway/HighwayRfClassifier.cpp( 324) extractorNames: [4]{EdgeDistanceRmseAggregator, EdgeDistanceSigma_Aggregator, hoot__AngleHistogramExtractor, hoot__WeightedMetricDistanceExtractor_way_agg__MeanAggregator_point_agg__RmseAggregator}
+11:44:47.356 INFO  ...ighway/HighwayRfClassifier.cpp( 325) missingExtractors: [0]{}
+11:44:47.356 INFO  ...ighway/HighwayRfClassifier.cpp( 326) _rfFactorLabels: [4]{EdgeDistanceRmseAggregator, EdgeDistanceSigma_Aggregator, hoot__AngleHistogramExtractor, hoot__WeightedMetricDistanceExtractor_way_agg__MeanAggregator_point_agg__RmseAggregator}
+11:44:47.387 INFO  ...ighway/HighwayMatchCreator.cpp(  96) neighbor counts, max: 2 mean: 1.125
+11:44:47.425 INFO  .../hoot/rules/LinearWaterway.js"(  33) "Automatically calculating search radius..."
+11:44:47.429 INFO  .../core/conflate/RubberSheet.cpp( 326) Found 6 potential ties.
+11:44:47.429 INFO  .../core/conflate/RubberSheet.cpp( 364) Found 2 confident ties.
+11:44:47.429 INFO  ...ate/SearchRadiusCalculator.cpp( 158) Calculated search radius = 15.97912392920059
+11:44:47.440 INFO  .../OptimalConstrainedMatches.cpp(  55) Calculating match conflicts
+11:44:47.449 INFO  .../OptimalConstrainedMatches.cpp(  57) Done calculating match conflicts
+11:44:47.449 INFO  ...h/GreedyConstrainedMatches.cpp(  68) Calculating match conflicts
+11:44:47.449 INFO  ...h/GreedyConstrainedMatches.cpp(  70) Done calculating match conflicts
+11:44:47.458 INFO  .../cpp/hoot/core/ops/NamedOp.cpp(  57) Applying operation: hoot::SuperfluousNodeRemover
+11:44:47.458 INFO  ...ops/SuperfluousNodeRemover.cpp(  54) Removing superfluous nodes...
+Reprojecting 0 / 153         
+11:44:47.459 INFO  .../cpp/hoot/core/ops/NamedOp.cpp(  57) Applying operation: hoot::SmallWayMerger
+11:44:47.459 INFO  .../cpp/hoot/core/ops/NamedOp.cpp(  75) Applying visitor: hoot::RemoveMissingElementsVisitor
+11:44:47.459 INFO  .../cpp/hoot/core/ops/NamedOp.cpp(  75) Applying visitor: hoot::RemoveInvalidReviewRelationsVisitor
+11:44:47.459 INFO  .../cpp/hoot/core/ops/NamedOp.cpp(  57) Applying operation: hoot::RemoveDuplicateReviewsOp
+11:44:47.459 INFO  .../cpp/hoot/core/ops/NamedOp.cpp(  57) Applying operation: hoot::BuildingOutlineUpdateOp
+11:44:47.459 INFO  .../cpp/hoot/core/ops/NamedOp.cpp(  57) Applying operation: hoot::AddHilbertReviewSortOrderOp
+Reprojecting 0 / 144         
+11:44:47.620 INFO  .../hoot/core/cmd/BaseCommand.cpp(  78) Loading map data from test-output/cmd/ConflateCmdTest/output.osm ...
+11:44:47.624 INFO  .../hoot/core/cmd/BaseCommand.cpp(  78) Loading map data from test-files/cmd/slow/ConflateCmdTest/output.osm ...
+11:44:47.628 WARN  ...core/scoring/MapComparator.cpp( 123) Tags do not match:
```

Comment:
Looks like there is a problem with this test.  There shouldn't be any difference between
`test-output/cmd/ConflateCmdTest/output.osm`
and
`test-files/cmd/slow/ConflateCmdTest/output.osm`
then the rest of this file, from here down, shouldn't exist.